Dockerfile

```diff
@@ -2,14 +2,16 @@
 FROM ubuntu:20.04 as builder 
 MAINTAINER Kevin Kreiser <kevinkreiser@gmail.com>
 
+ARG CONCURRENCY=$(nproc)
+
 # set paths
 ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH
 ENV LD_LIBRARY_PATH /usr/local/lib:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/lib32:/usr/lib32
 
 # install deps
 WORKDIR /usr/local/src/valhalla
-COPY ./scripts/install-linux-deps.sh /usr/local/src/valhalla/scripts
-RUN bash ./scripts/install-linux-deps.sh
+COPY ./scripts/install-linux-deps.sh /usr/local/src/valhalla/scripts/install-linux-deps.sh
+RUN bash /usr/local/src/valhalla/scripts/install-linux-deps.sh
```

Comment:
so just manually use bash, that should work if its about not being executable

Reply:
it was actually that docker copied the script and names it literally `/usr/local/src/valhalla/scripts`.. always getting confused with these docker copy files shite.. anyways, I could've it built it myself in the first PR :sweat_smile: